### ISSUE TITLE
[question] About official repo and feature

### ISSUE THREAD

**Issue by SalahAdDin** [skirsdeda/wagtail-translation#2]
Which the difference between this repository and the official?

Are you working in a way to display all language fields in different views? I think this is more confortable for work.

**Comment by skirsdeda** [comment]
What do you mean by "official"?

Could you elaborate on language fields in different views? How would it look?

**Comment by SalahAdDin** [comment]
The project base, from info-portugal.

More messy, not all fields in the same view page, if you have many language you will have many fields in the same page view, this will make more difficult his use.

**Comment by skirsdeda** [comment]
This is a clean reimplementation (no hardcoding, no repeating code, etc.) of what wagtail-modeltranslation provides. But the code is rather different, so this doesn't have much to do with wagtail-modeltranslation. It's a project on it's own.

You can have different languages in different tabs if you define ``content_panels`` on your Page submodels.

**Comment by SalahAdDin** [comment]
Did you reviewed the new `wagtail-modeltranslation` version?

**Comment by skirsdeda** [comment]
Yeah, the only improvement is that it doesn't copy entire django-modeltranslation project. Otherwise, same hardcoding and monkey-patching everything.

**Comment by SalahAdDin** [comment]
Yes, I saw it, I have problems with these package yet. 

How can manage each language fields in different tags, or better, in different views like `wagtailtranslation` or `wagtailtrans`?
